src/openvic-simulation/country/CountryInstance.hpp

```diff
@@ -227,6 +235,8 @@ namespace OpenVic {
 		bool set_ruling_party(CountryParty const& new_ruling_party);
 		bool add_reform(Reform const& new_reform);
 
+		bool set_value_for_slider(SliderManager& slider_manager, int const* new_value);
```

Comment:
Why point to an int here?
The new value can be passed by value, so `const int new_value`.